jms/org.wso2.carbon.transport.jms/src/main/java/org/wso2/carbon/transport/jms/factory/JMSConnectionFactory.java

```diff
@@ -437,18 +442,6 @@ public Destination getDestination(Session session) throws JMSConnectorException
     }
 
     /**
-     * Get a message consumer for particular session and destination.
-     *
-     * @param session     JMS Session to create the consumer
-     * @param destination JMS destination which the consumer should listen to
-     * @return Message Consumer, who is listening in particular destination with the given session
-     * @throws JMSConnectorException Thrown when creating jms message consumer.
-     */
-    public MessageConsumer getMessageConsumer(Session session, Destination destination) throws JMSConnectorException {
```

Comment:
This cannot be done in a patch release as we are removing a public method. This can only be added in a major release.

Reply:
Fixed with a31c7a4